.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+go:
```

Comment:
Sounds fine. For some reason I thought we said something about supporting 1.2+, but I guess that's not in a README, just from a commit fixing it up.